sphinx/io.py

```diff
@@ -88,6 +88,16 @@ def __init__(self, app, *args, **kwargs):
         self.transforms = self.transforms + app.registry.get_transforms()
         SphinxBaseReader.__init__(self, *args, **kwargs)  # type: ignore
 
+    def __init__(self, app, parsers={}, *args, **kwargs):
```

Comment:
Oh, my merging was wrong.  There are two `__init__()`s.

Reply:
@tk0miya apart from that I don't understand why ``class SphinxBaseReader(standalone.Reader)`` in master seems to lack code lines

https://github.com/sphinx-doc/sphinx/blame/master/sphinx/io.py

https://github.com/sphinx-doc/sphinx/blame/stable/sphinx/io.py

This may impact this merge too regarding ``parsers={}`` part.

Reply:
Refs: a12399dbe429f552ab012099491c577de668a81d

Reply:
Yes, it is intended changes. see #4294 in detail.